openml-h2o/pom.xml

```diff
@@ -141,13 +141,13 @@
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
             <!-- This version was specially chosen to match the one used by h2o-bindings -->
-            <version>3.3.1</version>
+            <version>4.6.0</version>
```

Comment:
Is there any way we can avoid the manual sync?

Reply:
I didn't try to understand what happened if the version is not enforced. Maybe this is not a problem anymore.